core/shared/src/main/scala/doodle/syntax/RasterSyntax.scala

```diff
@@ -0,0 +1,16 @@
+package doodle
```

Comment:
This file is missing it's header. It should be added if you run the `build` command from sbt, the `prePR` command, or the `headerCreateAll` command.